test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -64,12 +64,15 @@ if (cluster.isMaster) {
 
   socket1.bind({
     address: 'localhost',
-    port: common.PORT,
+    port: 0,
     exclusive: false
-  }, () => {
-    socket2.bind({ port: common.PORT + 1, exclusive: true }, () => {
+  }, common.mustCall(() => {
+    socket2.bind({
+      port: socket1.address().port + 1,
```

Comment:
I don't think this is correct. There's no guarantee that this port is in use by a different process. TBH I'm not sure there's a better solution than using `common.PORT`, maybe someone has a better idea.